controllers/materials/index.js

```diff
@@ -239,17 +206,17 @@ function initForm({ router, routeConfig }) {
                             delete req.session[sessionOrderKey];
                             delete req.session[sessionBlockedItemKey];
                             req.session.save(() => {
-                                renderForm(req, res, FORM_STATES.SUBMISSION_SUCCESS);
+                                renderForm({ req, res, status: FORM_STATES.SUBMISSION_SUCCESS });
                             });
                         });
                     })
                     .catch(err => {
                         Raven.captureException(err);
-                        renderForm(req, res, FORM_STATES.SUBMISSION_ERROR);
+                        renderForm({ req, res, status: FORM_STATES.SUBMISSION_ERROR });
                     });
             } else {
-                req.flash('formErrors', errors.array());
-                req.flash('formValues', req.body);
+                req.session['materials.formData'] = matchedData(req, { locations: ['body'] });
```

Comment:
Can't remember for certain but I think this will only return fields that were required?

Reply:
Seems to work as I'd expect it to here, I'm only using it to repopulate form fields, same way as `create-form-router`. Will double check though.